File: app/src/main/java/com/channa/mobiledatausageapp/data/response/DatastoreResponse.java
package com.channa.mobiledatausageapp.data.response;

public class DatastoreResponse {
    private String help;
    private Boolean success;
    private ResultResponse result;

    public DatastoreResponse() {
    }

    public DatastoreResponse(String help, Boolean success, ResultResponse result) {
        this.help = help;
        this.success = success;
        this.result = result;
    }

    public String getHelp() {
        return help;
    }

    public void setHelp(String help) {
        this.help = help;
    }

    public Boolean getSuccess() {
        return success;
    }

    public void setSuccess(Boolean success) {
        this.success = success;
    }

    public ResultResponse getResult() {
        return result;
    }

    public void setResult(ResultResponse result) {
        this.result = result;
    }
}

File: app/src/main/java/com/zak/datauses/data/response/DatastoreResponse.kt
package com.zak.datauses.data.response

class DatastoreResponse {
    var help: String? = null
    var success: Boolean? = null
    var result: ResultResponse? = null

    constructor() {}
    constructor(help: String?, success: Boolean?, result: ResultResponse?) {
        this.help = help
        this.success = success
        this.result = result
    }
}